DDR4 OC Guide.md

```diff
@@ -375,62 +397,82 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
 
 ## Finding a Baseline
-1. On Intel, start off with 1.15V VCCSA and VCCIO.  
-   On AMD, start off with 1.10V SOC.
+1. * Ensure your sticks are in the recommended DIMM slots (usually 2 and 4).
+   * Make sure your CPU overclock is disabled when tuning RAM, as an unstable CPU can lead to memory errors. Likewise, when pushing high frequency with tight timings, your CPU may become unstable and may need to be re-done.
+   * Make sure your UEFI/BIOS is up to date.
+
+2. On Intel, start with 1.15V VCCSA and VCCIO.  
+   On AMD, start with 1.10V SOC.
    * SOC voltage might be named differently depending on the manufacturer.
      * Asrock: CPU VDDCR_SOC Voltage. If you can't find that, you can use SOC Overclock VID hidden in the AMD CBS menu.
        * [VID values](https://www.reddit.com/r/Amd/comments/842ehb/asrock_ab350_pro4_guide_bios_overclocking_raven/).
      * Asus: VDDCR SOC.
      * Gigabyte: (Dynamic) Vcore SOC.
-       * Note that dynamic Vcore SOC is an offset voltage. The base voltage can change automatically when increasing DRAM frequency. +0.100V at DDR4-3000 might result in 1.10V actual, but +0.100V at DDR4-3400 might result in 1.20v actual.
+       * Note that dynamic Vcore SOC is an offset voltage. Therefore, the base voltage can change automatically when increasing DRAM frequency. For example, +0.100V at DDR4-3000 might result in 1.10V actual, but +0.100V at DDR4-3400 might result in 1.20v actual.
      * MSI: CPU NB/SOC.
-2. Set DRAM voltage to 1.40V. If you're using ICs that roll over above 1.35V, set 1.35V.
+3. Set DRAM voltage to 1.40V. If you're using ICs that roll over above 1.35V, set 1.35V.
    * "Roll over" means that the IC becomes more unstable as you increase the voltage, sometimes to the point of not even POSTing.
-   * ICs that are known to roll over above 1.35V include but is not limited to: 8Gb Samsung C-die, older Micron/SpecTek ICs (before 8Gb Rev. E).
-3. Set primary timings to 16-20-20-40 (tCL-tRCD-tRP-tRAS) and tCWL to 16.
-   * Most ICs need loose tRCD and/or tRP which is why I recommend 20.
+   * ICs that are known to roll over above 1.35V include but are not limited to: 8Gb Samsung C-die, older Micron/SpecTek ICs (before 8Gb Rev. E).
+   * Here is a list of common ICs and safe voltages for them:
+   
+   | IC                                | Daily voltage (V) | Extreme voltage |
+   | :---:                             | :---------:       | :-------------: |
+   | 8Gbit Samsung B-Die               | 1.45 - 1.55       | 1.55 - 1.65     |
+   | 8Gbit Hynix DJR/16Gbit AJR        | 1.45 - 1.55       | 1.50 - 1.65     |
+   | 8Gbit Micron Rev. E/16Gbit Rev. B | 1.45 - 1.55       | 1.55 - 1.60     |
+   | 8Gbit Hynix CJR                   | 1.35 - 1.40       | 1.40 - 1.45     |
+   | 16Gbit Hynix CJR                  | 1.40 - 1.50       | 1.50 - 1.55     |
+   | 8Gbit Nanya B-Die                 | 1.35 - 1.40       | 1.40 - 1.45     |
+   | Samsung C-Die                     | 1.30 - 1.35       | Likely Unsafe   |
```

Comment:
This should go under Maximum Recommended Daily Voltage

Clarify what extreme voltage means

Reply:
That makes sense, I will also add a link to the section in the part where they overclock.

Reply:
This has been moved and clarified.